internal/errcheck.go

```diff
@@ -63,6 +64,21 @@ func (e byName) Less(i, j int) bool {
 	return ei.line < ej.line
 }
 
+// findPackage is similar to the default implementation (loader.defaultFindPackage),
+// but allows local imports like "./foo" to resolve relative to the current directory.
+func findPackage(ctxt *build.Context, path string) (*build.Package, error) {
+	wd, err := os.Getwd()
```

Comment:
I think there's a chance it'd be a worthwhile optimization to factor out the `os.Getwd()` call outside of this func.

Unless I'm mistaken, cwd cannot change during the execution of the process (unless the process itself changes it), so it's enough to get cwd once at init.